packages/contracts-bsc/contracts/BscTokenController.sol

```diff
@@ -533,10 +534,10 @@ contract BscTokenController {
     }
 
     /**
-     * @dev Claim ownership of an arbitrary HasOwner contract
-
+     * @dev Claim ownership of an arbitrary IHasOwner contract
+     */
     function issueClaimOwnership(address _other) public onlyOwner {
-        HasOwner other = HasOwner(_other);
+        IHasOwner other = IHasOwner(_other);
         other.claimOwnership();
```

Comment:
I assume this function exists to claim impl ownership of `token`? If so, then I have some suggestions to improve the original code:
- for consistency with the `proxy functions` section above, I would rename this function to `claimTrueCurrencyImplOwnership()` and move it into its own section immediately below `proxy functions`:
```
    /*
    ========================================
    impl functions
    ========================================
    */
```
- drop the `address _other` argument. There's no need to support other arbitrary addresses if the controller will only ever claim `token` impl ownership. Ideally, the body of this function should just be `token.claimOwnership()`.
- no other functions in `BscTokenController.sol` call this, so this can safely be marked `external`.

I would also suggest making these changes in a separate PR so unrelated changes can be merged.